app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class UserMailer < ApplicationMailer
+  default from: "notifications@example.com"
+
+  def notify_post_owner(comment:)
+    @comment = comment
+    @user = comment.post.user
+
+    mail(to: @user.email, subject: "[#{ENV.fetch('APP_NAME')}] - A new comment has been posted")
```

Comment:
I'd provide more information in the subject, such as "{APP_NAME} - New comment on {POST_NAME}". This may result in a long subject, but I believe it's worth it.